networkx/algorithms/tree/branchings.py

```diff
@@ -76,6 +76,32 @@ def branching_weight(G, attr="weight", default=1):
     """
     Returns the total weight of a branching.
 
+    You must access this function through the networkx.algorithms.tree module.
+
+    Parameters
+    ----------
+    G : DiGraph
+        The directed graph.
+    attr : str
+        The attribute to use as weights. If None, then each edge will be
+        treated equally with a weight of 1.
+    default : float
+        When `attr` is not None, then if an edge does not have that attribute,
+        `default` specifies what value it should take.
+
+    Returns
+    -------
+    weight: int or float
+        The total weight of the branching.
+
+    Examples
+    --------
+    >>> from networkx.algorithms import tree
+    >>> G = nx.DiGraph()
+    >>> G.add_weighted_edges_from([(0, 1, 2), (1, 2, 4), (2, 3, 3), (3, 4, 2)])
+    >>> tree.branching_weight(G)
```

Comment:
It's actually easier to get to that using an import.
`nx.tree.branching_weight(G)` should work. Sorry I didn't catch that before... :}

Reply:
No problem!! I updated it.